### ISSUE TITLE
Why the loss function is not same with paper said?

### ISSUE THREAD

**Issue by AlexFEIII** [omarsayed7/Deep-Emotion#14]
hello, I'm a student who is learning AI.
I was download your code, but I have a problem about this code, 
The paper saids "The loss function in this work is simply the summation of two terms, the classification loss(cross-entropy), and the regularization term(which is l2 norm of the weights in the last two fully-connected layers)"
In the main.py, I just see "the classification loss". Is there any particular reason for this?

finally, thank you for your code contributions

**Comment by hnhoangdz** [comment]
l2 norm is the loss function, it is a regularization technique. in this paper, the author wanted to applied l2 norm for only last fcn layers
